### PR TITLE
ionic: add page

### PR DESCRIPTION
**Pull request by BinaryBrain**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/ionic.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+`ionic g {{page}}`
+
+- Show versions of ionic, cordova, environment, etc.:
+
+`ionic info`
+
+- Run app on an android/ios device:
+
+`ionic cordova run {{android}} --device`
```
Comment (agnivade) [inline pages/common/ionic.md]: This can be `android|ios`

File: pages/common/ionic.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# ionic
+
+> Ionic project manager and builder.
```
Comment (agnivade) [inline pages/common/ionic.md]: We should clarify what this is - "A framework to build hybrid mobile apps"

Comment (agnivade) [reply pages/common/ionic.md]: I meant to replace the line with this sentence. Sorry if it wasn't clear. Generally, we avoid using the command name again in the heading. So just "project manager and builder" is not unambiguous enough. Hence my suggestions.

Comment (BinaryBrain) [reply pages/common/ionic.md]: Oh sorry, I see what you meant now.

**Pull request merged**